### DISCUSSION TITLE
More feature for AR Goggles

### DISCUSSION THREAD

**Discussion by YeahBoi321** [IntelligenceModding/AdvancedPeripherals#146] [Ideas]
I believe that either AR goggles should have an output as well as an input, such as rotational data of the player, or player velocity/speed, not giving direct coordinate input, just vectors, or there should be an upgraded version of the AR goggles, such as VR goggles, that give said outputs back to the controller if requested.

**Comment by SirEndii** [comment]
The ar goggles with get a major rework in 0.8r(or maybe 1.0r) in 1.18

**Reply by SirEndii** [reply]
And sorry for the *very* late answer. Didn't know this exists.

**Reply by YeahBoi321** [reply]
Sorry for the _very_ late reply. Thanks! And good to know that we will be able to do more with them than just write to the screen! Or at least, I hope.

**Reply by SirEndii** [reply]
Some features are implemented
Like receiving rotational data

**Reply by zayzero** [reply]
100%

**Reply by YeahBoi321** [reply]
> Some features are implemented Like receiving rotational data

So many late replies, this one is really bad. Is any documentation updated with this data?
